lib/ansible/modules/remote_management/manageiq/manageiq_dynamic_resource_definition.py

```diff
@@ -35,7 +35,16 @@
   properties:
     description:
       - The dynamic resource definition's properties.
-    version_added: '2.6'
+    suboptions:
+        attributes:
+          description: dictionary of attributes to pass to the API.
+        associations:
+          description: dictionary with service and provider class.
+        methods:
+          description: list of methods to add to dynamic resource definition.
+  manageiq_connection:
+    description: ManageIQ connection details.
```

Comment:
ooh, try removing above 2 lines and see if it makes CI happier.
 `manageiq_connection` suboptions (including `verify_ssl`) documentation should be inherited via `extends_documentation_fragment: manageiq`, and having this here might interfere with that...